README.md

```diff
@@ -56,3 +56,28 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 1. Create a Trello Account.
 2. Grab your API Key and Token by following [these instructions](https://trello.com/app-key).
 3. Store your API Key & Token in the .env file
+
+## Ansible: SSH into your Managed Node
+
+To connect to the Managed Node via ssh, use the command:
```

Comment:
I _think_ you're meaning to refer future users to SSH into the control node rather than the managed node - does that sound right?